scripts/create-component/index.js

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+
+const inquirer = require('inquirer')
+const copyTemplateDir = require('copy-template-dir')
+
+const { getPackages, toPackageName } = require('../utils/packages')
+const { smush, pascalize } = require('../utils/string')
+
+const MINIMUM_REACT_VERSION = '^16.2.0'
+
+const packagesDir = path.resolve(__dirname, '..', '..', 'packages')
+const templateDir = path.resolve(__dirname, 'package-template')
+
+const copyPackageFromTemplateDir = (source, dest, vars) => {
+  return new Promise((resolve, reject) => {
+    copyTemplateDir(source, dest, vars, (err, createdFiles) => {
+      if (err) reject(err)
+      else resolve(createdFiles)
+    })
+  })
+}
+
+async function main() {
+  const packages = await getPackages()
+  const packagesMap = packages.reduce((acc, pkg) => ({
+    ...acc,
+    [pkg.name]: pkg.version
+  }))
+
+  const suggestedDependencies = [
+    'button',
+    'core',
+    'icon',
+    'normalize',
+    'theme',
+    'util'
```

Comment:
So few things use this, and I think I'd rather component devs not try to get too fancy about splitting out reusable things.  I think we as product owners are probably better equipped with a system vision to do that once we have a working component added to the library.

Reply:
i will remove - this list is mostly a shot in the dark. let me know if there is anything that would be valuable here